Source/WebCore/platform/graphics/WidthIterator.cpp

```diff
@@ -187,7 +187,9 @@ bool WidthIterator::hasExtraSpacing() const
 
 static void resetGlyphBuffer(GlyphBuffer& glyphBuffer, GlyphBufferStringOffset index)
 {
+#if USE(CG)
     ASSERT(index >= 0);
+#endif
     auto unsignedIndex = static_cast<unsigned>(index);
```

Comment:
Also this cast wouldn't be needed if not using CG.

Reply:
But doesn't hurt?